app/services/unified_chat_service.py

```
@@ -0,0 +1,1261 @@
+
+Provide insights on these real opportunities."""
+        
+        # Default prompt for other intents
+        return f"""User asked: "{message}"
+        
+Intent: {intent.value}
+Available Data: {list(context_data.keys())}
+
+Provide a helpful response using the real data available. Never use placeholder data."""
+    
+    async def _get_user_config(self, user_id: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `user_id`

(ARG002)